frontend-service/src/components/TopBarGame.tsx

```diff
@@ -1,16 +1,23 @@
 import React from 'react'
```

Comment:
Pretty much all the changes in this file are from type improvements are downstream of consolidating all those `useState`s in Game.tsx.